packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -141,6 +142,13 @@ export const render = async (req, res, next) => {
         })
     }
 
+    // Serialize the routes and add them to the config. We'll use this on the client-side later.
+    config.app.routes = routes.map((route) => ({
+        path: route.path,
+        componentName: route.component.displayName.match(/\((\w+)\)/)[1],
+        componentProps: route.props
+    }))
```

Comment:
Based off our conversation we had yesterday:

We look to be serializing all the routes exported from the applications routes.jsx file. Because this list is very long after being processed via the `configureRoutes` function, it will end up increasing the size of the html. 

In our project we have about 15 routes and because of our sites/locales configuration ends up being closer to 350 routes. We can imagine that our customers will have not only more routes but more sites and locales.. meaning that they could potentially have thousands of routes being serialized. This could slow things down. 

We should test this and make sure we aren't affecting performance too much.

Reply:
Additionally we are serializing a routeCongig object using it's path, componentName, componentProps. 

This was a solution that I whipped up quick and dirty, we should spend sometime to see if there are any better solutions out there.

Reply:
A possible solution is to only serialize the one route that we get from the seo url mapping api instead of all the routes. It's a little more narrow in it's implementation, but it should work.